data/met/262098.json

```diff
@@ -0,0 +1,221 @@
+{
+  "@context": "https://linked.art/ns/v1/linked-art.json",
+  "id": "https://collectionapi.metmuseum.org/public/collection/v1/objects/262098",
+  "type": "HumanMadeObject",
+  "identified_by": [
+    {
+      "id": "http://lod.example.org/museum/Identifier/46",
+      "type": "Identifier",
+      "content": 262098,
+      "classified_as": [
+        {
+          "id": "http://vocab.getty.edu/aat/300404621",
+          "type": "Type",
+          "_label": "Local Number"
+        }
+      ]
+    },
+    {
+      "id": "http://lod.example.org/museum/Identifier/47",
+      "type": "Identifier",
+      "content": "1977.657.1",
+      "classified_as": [
+        {
+          "id": "http://vocab.getty.edu/aat/300312355",
+          "type": "Type",
+          "_label": "Accession Number"
+        }
+      ]
+    },
+    {
+      "id": "http://lod.example.org/museum/Name/118",
+      "type": "Name",
+      "content": "[Looking from Garage to Studio Door, Abiquiu, New Mexico]",
+      "classified_as": [
+        {
+          "id": "http://vocab.getty.edu/aat/300195168",
+          "type": "Type",
+          "_label": "Title"
+        }
+      ]
+    }
+  ],
+  "produced_by": {
+    "id": "http://lod.example.org/museum/Production/23",
+    "type": "Production",
+    "timespan": {
+      "id": "http://lod.example.org/museum/TimeSpan/69",
+      "type": "TimeSpan",
+      "begin_of_the_begin": 1957,
+      "end_of_the_end": 1958,
+      "identified_by": [
```

Comment:
This looks a bit strange compared to the other data we've been producing. Is an integer the expected encoding of dates comprised of just a year?

Reply:
Agree, should be full dateTime - lazy programmer.